code/data_structures/src/list/singly_linked_list/menu_interface/link.h

```diff
@@ -0,0 +1,397 @@
+#include <stdio.h>
```

Comment:
Please follow our C style guide (you can find it from the root directory/guides/c/README.md)

Reply:
I have changed the code in both files according to C coding style mentioned in above file. Kindly see it.